isaric/parsers/ccp-drc.toml

```diff
@@ -253,10 +271,10 @@
 [visit.icu_admission_dates]
   combinedType = "set"
   excludeWhen = "none"
-  fields = [
-    { field = "date_icu_adm", description = "Date of ICU Admission:" },
-    { field = "overall_icu_hostdat", description = "Date of ICU Admission:" },
-  ]
+  fields = [{ field = "date_icu_adm", description = "Date of ICU Admission:" }]
+
+[visit.pathogen_test_date]
+  field = "results_date"
```

Comment:
@pipliggins I think it best to use sample_date here, as for most instances across the datasets it is the date of the sample that is reported, rather than the date that the results were received or reviewed